.env.example

```
@@ -2,3 +2,4 @@
 # Copy to home-manager/.env and provide real values. The .env file stays local.
 MY_SECRET=replace-me
 # GITHUB_TOKEN=ghp_your_token_here
+CLIPROXY_MANAGEMENT_PASSWORD=your-management-key-here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLIPROXY_MANAGEMENT_PASSWORD key should go before the MY_SECRET key

(UnorderedKey)